CONTRIBUTING.md

```diff
@@ -18,16 +18,31 @@ If you find any existing issues that you are interested in fixing, you are welco
 
 Getting setup is quick and easy. Follow the steps below to get your your dev environment up.
 
-1. Clone GitHub repo
+1. Fork GitHub repo
 2. Install dependencies
-3. Build and link packages
+3. Build packages
 
 ```
-$ git clone git@github.com:amplitude/Amplitude-TypeScript.git
+$ git clone <HTTPS_OR_GIT_URL>
 $ yarn
 $ yarn build
 ```
 
+This repo contains mutliple major versions of all packages. For contributions to version `1.x`, create a branch off `v1.x`. For contributions to the `2.x` (latest) version, create a branch off `main`. Refer to the table below for more infomation about the release status of each package.
+
+|Package|Version|Status|Dev Branch
```

Comment:
Missing an end `|`?

Reply:
i think it still works without but i can add

Reply:
```suggestion
|Package|Version|Status|Dev Branch|
```

Reply:
Thanks.